GOVERNANCE.md

```diff
@@ -0,0 +1,193 @@
+    * Participating in, and leading, community meetings
+* Requirements
+    * Experience as a Maintainer for at least 3 months
+    * Demonstrates a broad knowledge of all components, repositories, and subprojects of the Podman project.
+    * Is able to exercise judgment for the good of the project, independent of their employer, friends, or team
+    * Mentors new Maintainers and Core Maintainers
+    * Able to make decisions and contributions affecting the whole project, including multiple subprojects and repositories
+    * Can commit to maintaining a high level of contribution to the project as a whole
+* Additional privileges:
+    * Represent the project in public as a senior project member
+    * Communicate with the CNCF on behalf of the project
+    * Have a vote in Core Maintainer decision-making meetings
+
+#### Process of becoming a Core Maintainer Member:
+1. A current maintainer must be sponsored by Core Maintainer. The Core Maintainer will open a PR against the main Podman repository and add the nominee as a Core Maintainer in the MAINTAINERS.md file.
+2. The nominee will add a comment to the PR confirming that they agree to all requirements and responsibilities of becoming a Core Maintainer.
+3. A majority of the current Core Maintainers must then approve the PR.
+4. If, for some reason, all existing members are inactive according to the Inactivity policy below or there are no Core Maintainers due to resignations, a supermajority vote of maintainers can bypass this process and approve new Core Maintainers directly.
+
+### Community Manager
+Community managers are responsible for the project’s community interactions, including project social media, website maintenance, gathering metrics, managing the new contributor process, ensuring documentation is easy to use and welcoming to new users, and managing the project’s interactions with the CNCF. This is a nontechnical role, and as such does not require technical contribution to the project.
+
+* Responsibilities include:
+    * Participating in CNCF maintainer activities
+    * Arranging, participating in, and leading, community meetings
+    * Managing the project website and gathering associated metrics
+    * Managing the project’s social media accounts and mailing lists and gathering associated metrics
+* Requirements
+    * Sustained high level of contribution to the community, including attending and engaging in community meetings, contributions to the website, and contributions to documentation, for at least six months
+    * Is able to exercise judgment for the good of the project, independent of their employer, friends, or team
+    * Can commit to maintaining a high level of contribution to the project's community, website, and social media presence
+    * Advocates for the community in Maintainer and Core Maintainer meetings
+* Additional privileges:
+    * Represent the project in public
+    * Communicate with the CNCF on behalf of the project
```

Comment:
Do you want just "Communicate" with the CNCF or do you want the ability to represent on behalf of the project? 

For example do you want this person to be able to ask for resources, do event coordination with us, submit maintainer CFPs, be able to file support tickets on behalf of other maintainers, access to #maintainers-circle, etc. 

If so I'd strengthen this a tad so we know to give them the proper rights.

Reply:
I was aiming for the same rights as maintainers, so I'll use the same wording we use for them. Good catch.